src/operator/mergeMapTo.ts

```diff
@@ -112,7 +112,7 @@ export class MergeMapToSubscriber<T, I, R> extends OuterSubscriber<T, I> {
     }
   }
 
-  private _innerSub(ish: any,
+  private _innerSub(ish: SubscribableOrPromise<I>,
```

Comment:
@david-driscoll 

Okay. I fixed in https://github.com/ReactiveX/rxjs/pull/1749/commits/1492043d2630c378cb0869ed27268b4eff5d945b